modules/face/data/mean_shape.txt

```diff
@@ -0,0 +1,195 @@
+137.856,219.266
```

Comment:
But then this is somewhere hardcoded, to which bounding box this relates.
However, I do not get why this would be overkill. Taking x/widthBox and y/heightBox gives you a parameter that is already procentual to all feature boxes?